src/data/api.js

```diff
@@ -5,14 +5,26 @@ import { ExamAction } from '../constants';
 const BASE_API_URL = '/api/edx_proctoring/v1/proctored_exam/attempt';
 
 export async function fetchExamAttemptsData(courseId, sequenceId) {
-  const url = new URL(
-    `${getConfig().LMS_BASE_URL}${BASE_API_URL}/course_id/${courseId}`,
-  );
-  if (sequenceId) {
-    url.searchParams.append('content_id', sequenceId);
+  let data;
+  if (!getConfig().EXAMS_BASE_URL) {
+    const url = new URL(
+      `${getConfig().LMS_BASE_URL}${BASE_API_URL}/course_id/${courseId}`,
+    );
+    if (sequenceId) {
+      url.searchParams.append('content_id', sequenceId);
+    }
+    url.searchParams.append('is_learning_mfe', true);
+    const urlResponse = await getAuthenticatedHttpClient().get(url.href);
+    data = urlResponse.data;
```

Comment:
I had to modify this to no longer unpack the data value because of how that variable was scoped. In order to not return within each conditional, I defined the variable prior to the conditional, modify within each branch, and then return after